src/modules/AdvancedPaste/AdvancedPaste/Helpers/JsonHelper.cs

```diff
@@ -33,6 +34,19 @@ internal static class JsonHelper
         private static readonly Regex CsvRemoveStartAndEndQuotationMarksRegex = new Regex(@"^""(?=(""{2})+)|(?<=(""{2})+)""$");
         private static readonly Regex CsvReplaceDoubleQuotationMarksRegex = new Regex(@"""{2}");
 
+        private static bool IsJson(string text)
+        {
+            try
+            {
+                JsonDocument.Parse(text);
```

Comment:
using JsonDocument doc = JsonDocument.Parse(text);

Looks like this function return something should be disposed after using.